docs/UserGuide/IoTDB-SQL-Language/DML-Data-Manipulation-Language.md

```diff
@@ -207,7 +207,7 @@ IoTDB supports the 'order by time' statement since 0.11, it's used to display re
 For example, the SQL statement is:
 
 ```sql
-select * from root.ln where time > 1 order by time desc limit 10;
+select * from root.ln.* where time > 1 order by time desc limit 10;
```

Comment:
The exampled SQL statements in IoTDB Doc seem to explain operations in a standard four layer scenario, root-sg-entity-measurement. If so, * or ** in this sql may bring same resultset. 
Should we add a scenario declaration in our DDL、DML and SQL docs? I think this may make it more understandable and reduce ambiguity.

Reply:
Fixed.